ParkPlanner/Park.cs

```diff
@@ -51,10 +51,10 @@ public partial class Datum
         public string Description { get; set; }
 
         [JsonProperty("latitude")]
-        public string Latitude { get; set; }
+        public double Latitude { get; set; }
```

Comment:
.Use Strong Types for Latitude and Longitude:

Instead of using string for Latitude and Longitude in the Datum class, consider using appropriate numeric types like double if these values are meant to represent geographical coordinates.

Reply:
1) I'd keep this as it were.  Double has floating point arithmetic issues that can be problematic with location data.
2) This was covered in a previous code review.  When that occurs, first in wins.